src/fileActions.jsx

```diff
@@ -237,3 +248,80 @@ export const RenameItemModal = ({ path, setPath, selected }) => {
         </Modal>
     );
 };
+
+export const CreateLinkModal = ({ currentPath, files, selected, errorMessage }) => {
+    const Dialogs = useDialogs();
+    const [originalName, setOriginalName] = useState(selected?.name || "");
+    const [newName, setNewName] = useState("");
+    const [type, setType] = useState("symbolic");
+
+    const createLink = () => {
+        const options = { err: "message", superuser: "try" };
+        cockpit.spawn(["ln", ...(type === "symbolic" ? ["-s"] : []), currentPath + originalName, currentPath + newName], options)
+                .then(Dialogs.close, (err) => {
+                    Dialogs.show(
```

Comment:
Please use the same dialog insteaf od opening new one and show all the previous dialog content + the error message.